homework01/rudolf/vector.cpp

```diff
@@ -0,0 +1,46 @@
+#include "vector.h"
+
+Vector::Vector()
+{
+	mX=0;
+	mY=0;
+}
+
+Vector::Vector(double x, double y): mX(x), mY(y){}
```

Comment:
You could call this constructor from the other one, see delgating constructor: https://en.cppreference.com/w/cpp/language/initializer_list#Delegating_constructor